Resources/Prototypes/_White/Entities/Mobs/NPCs/headcrab.yml

```
@@ -0,0 +1,145 @@
+# BASE
+
+- type: entity
+  parent:
+    - BaseMob
+    - MobDamageable
+    - MobCombat
+    - MobAtmosStandard
+    - MobBloodstream
+    - MobFlammable
+  id: BaseMobHeadcrab
+  abstract: true
+  name: headcrab
+  description: It looks like something parasitic.
+  components:
+  - type: GhostTakeoverAvailable
+  - type: GhostRole
+    allowSpeech: false
+    name: ghost-role-information-headcrab-name
+    description: ghost-role-information-headcrab-description 
+    raffle:
+      settings: short
+  - type: Sprite
```

```suggestion
  - type: Sprite
    offset: 0, 0.2
```
Add some Y offset to the sprite, Their actual sprite is below their fixture so they can "phase" themselves into a wall to the south of them.